vars/withDevopstools.groovy

```diff
@@ -3,7 +3,10 @@ def call(args=[:]) {
         usernamePassword(credentialsId: 'github-vega-ci-bot-artifacts', passwordVariable: 'TOKEN', usernameVariable:'USER')
     ]) {
         dir('devopstools') {
-            sh "go run main.go ${args.command} --network ${env.NET_NAME} --github-token \${TOKEN}"
+            sh (
+                script: "go run main.go ${args.command} --network ${env.NET_NAME} --github-token \${TOKEN}",
+                returnStdout: args.returnStdout ?: false,
+            )
```

Comment:
do we need to add `return` ?

Reply:
We might do it for visibility but In groovy last statement is always returned. You can't return nothing